demisto_sdk/commands/split/ymlsplitter.py

```diff
@@ -54,6 +57,7 @@ def __init__(
         input: str,
         output: str = "",
         file_type: str = "",
+        loaded_data: dict | None = None,
```

Comment:
if you add here predefined loaded_data, why do we need the `input_file` to be required?

Reply:
I'm assuming you mean `input`? It seems to be used for determining the output path:
`self.output = (Path(output) if output else Path(self.input.parent)).resolve()`

I also want to add this parameter without breaking BC and potentially breaking stuff in other places.